### PR TITLE
Color scheme updated, selected color feature added

### PR DESCRIPTION
**Pull request by estraily-navisjon**
I adjusted the color scheme for the disabled buttons, and updated the selected buttons to sudden death, and the colors for both new score and penalty add/edit

### CODE REVIEW

File: uwh-refbox/src/style.css [hunk 1]
```diff
@@ -57,9 +59,9 @@ button:hover {
     -gtk-icon-shadow: none;
     -gtk-icon-effect: none;
 }
-button:disabled {
-    background-color: @grayed_out_gray;
-    color: @dark_gray;
+button*:disabled {
+    background-color: @disabled_gray;
+    color: @text_disabled_gray;
```
Comment (TristanDebrunner) [inline uwh-refbox/src/style.css]: This sets all descendants of the button to also have this style (due to the missing `.`). In this case that makes the labels in the +60, etc. buttons also have a background color, which makes it look as if the buttons no longer have rounded corners. This fixes:
```button:disabled,
button.little-blue:disabled {
    background-color: @disabled_gray;
    color: @text_disabled_gray;
}
button:disabled label {
    color: @text_disabled_gray;
}
```

File: uwh-refbox/src/style.css [hunk 2]
```diff
@@ -267,6 +268,10 @@ button.gray:disabled {
 
 
 
+label.*:disabled {
+    background-color: @disabled_gray;
+    color: @text_disabled_gray;
+}
```
Comment (TristanDebrunner) [inline uwh-refbox/src/style.css]: The `.*:` syntax doesn't seem to work here, I think we need to fix as discussed:
```
label:disabled,
label.edit-parameter-header:disabled {
```

File: uwh-refbox/src/style.css [hunk 3]
```diff
@@ -306,14 +311,22 @@ label.edit-parameter-header {
     font-weight: bold;
     border-radius: 5px;
 }
+label.edit-parameter-header:disabled {
+    background-color: @disabled_gray;
+    color: @text_disabled_gray;
+}
```
Comment (TristanDebrunner) [inline uwh-refbox/src/style.css]: With above change this is unnecessary.

File: uwh-refbox/src/main.rs [hunk 4]
```diff
@@ -453,7 +490,39 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
...
+        let penalty_white_select_ = penalty_white_select.clone();
+
+        /*
+
+        let penalty_1min_ = penalty_1min.clone();
+        let penalty_2min_ = penalty_2min.clone();
+        let penalty_5min_ = penalty_5min.clone();
+        let penalty_dismiss_ = penalty_dismiss.clone();
+
+        */
```
Comment (TristanDebrunner) [inline uwh-refbox/src/main.rs]: remove

Comment (estraily-navisjon) [reply uwh-refbox/src/main.rs]: that's just where I was when at when I pushed it, I can re-create easily if you want it deleted

File: uwh-refbox/src/style.css (hunk 3 above)
Comment (estraily-navisjon) [reply uwh-refbox/src/style.css]: got it

File: uwh-refbox/src/style.css (hunk 2 above)
Comment (estraily-navisjon) [reply uwh-refbox/src/style.css]: ok

File: uwh-refbox/src/style.css (hunk 1 above)
Comment (estraily-navisjon) [reply uwh-refbox/src/style.css]: ok

Comment (TristanDebrunner) [reply uwh-refbox/src/style.css]: Your fix still doesn't solve the problem. Please implement above

Comment (estraily-navisjon) [reply uwh-refbox/src/style.css]: I did what you typed above, didn't fix the corner issue

Comment (estraily-navisjon) [reply uwh-refbox/src/style.css]: played around a little bit and got it working

**Pull request merged**